javascript/lastfm.js

```diff
@@ -34,7 +34,7 @@ async function fetchPlayData() {
   if (!username) {
     somethingWentWrong("No username provided");
     console.error("No username provided");
-    return;
+    resetToFirstState();
```

Comment:
Why resetToFirstState() here? You are just wasting computational power and bandwidth by loading the page again.